notebooks/china_data_combine.ipynb

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
<p>Could you clear the notebook outputs so there are no errors outputted ^</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>